resources/measures/upgrade_hvac_add_heat_pump_rtu/measure.rb

```diff
@@ -817,9 +1200,16 @@ def run(model, runner, user_arguments)
       end
       ### Cooling
       # define cooling stages; 40% to 100%, equally spaced; fractions from ResStock Reference file
+      # overriding 0.67 (value for stage3) to 0.61 (average of standard performance products) for modeling standard performance
```

Comment:
Is the 0.61 average the ratio of compressor sizes? So stage 1 is ~60% of total capacity and stage 2 is the remaining 40%? I don't see this value in the documentation.

Reply:
- changed 61% to 60% reflecting the addition of lennox products
- description added in measure doc